app/policies/draw_policy.rb

```diff
@@ -9,6 +9,10 @@ def update?
     user.rep? || user.admin?
   end
 
+  def start?
+    user.admin?
```

Comment:
changing this to `user.admin? && record.draft?` eliminates the need for `DrawHelper`